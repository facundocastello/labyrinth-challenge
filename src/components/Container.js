import styled from 'styled-components';

export default styled.div`
  margin: 20px;
  padding: 20px;
  width: fit-content;
  display: flex;
  align-items: center;
  background-color: black;
  border-radius: 10px;
`;