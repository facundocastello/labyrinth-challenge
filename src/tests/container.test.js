import React from 'react';
import Container from '../components/Container';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

describe( 'Container Block', () => {
  it( 'renders Container', () => {
    const tree = renderer.create(<Container />).toJSON();
    expect( tree ).toMatchSnapshot();
  } );
} );