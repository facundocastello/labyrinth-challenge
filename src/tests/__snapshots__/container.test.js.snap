// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Block renders Container 1`] = `
.c0 {
  margin: 20px;
  padding: 20px;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: black;
  border-radius: 10px;
}

<div
  className="c0"
/>
`;
